floris/simulation/wake_velocity/jensen.py

```diff
@@ -126,24 +122,32 @@ def function(
         """
 
         # Numexpr - do not change below without corresponding changes above.
-        dx = ne.evaluate("x - x_i")
-        dy = ne.evaluate("y - y_i - deflection_field_i")
-        dz = ne.evaluate("z - z_i")
+        # dx = ne.evaluate("x - x_i")
+        # dy = ne.evaluate("y - y_i - deflection_field_i")
+        # dz = ne.evaluate("z - z_i")
+
+        dx = x - x_i
+        dy = y - y_i - deflection_field_i
+        dz = z - z_i
 
```

Comment:
Just flagging the commenting out of NumExpression again, but ignore if this was intentional.